agent/pkg/k8s/policy.go

```diff
@@ -0,0 +1,65 @@
+package k8s
+
+import (
+	"context"
+
+	pb "github.com/buoyantio/linkerd-buoyant/gen/bcloud"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// SazGVR is the GroupVersionResource for the ServerAuthorization resource.
```

Comment:
can you mention this must stay in sync with https://github.com/linkerd/linkerd2/blob/22bbdad4d698cb327c4859befc3de0ddba51dfa4/pkg/k8s/policy.go#L24-L29, and also link to linkerd/linkerd2#6970 ?

going a step further, can we just import the `SazGVR` and `ServerGVR` vars from linkerd directly? (may require upgrading our linkerd2 dependency)